CMakeLists.txt

```diff
@@ -45,6 +43,11 @@ set(LCT_SRCS
 
 add_library(luacpptemplater STATIC ${LCT_SRCS})
 
+target_compile_definitions(
+    luacpptemplater
+    PRIVATE _GLIBCXX_USE_CXX11_ABI=1
```

Comment:
When C++11 support was finalized in gcc, they shipped a new ABI to conform with the new standard (e.g. std::strings were not permitted to have copy-on-write anymore). This might have been just my system, but despite using std=c++11 it compiled with the old ABI, so I added this. This won't cause any problems where the compiler would default to that anyway or different compilers.